content/departments/sales/tools/referral_and_reseller_policy.md

```diff
@@ -0,0 +1,85 @@
+**<span style="text-decoration:underline;">Referral Program Summary</span>**
```

Comment:
Headings in the handbook must be marked up as actual headings, rather than just visually bolded/underlined/etc. That makes them appear in the table of contents on the right of the page with the right hierarchy, is important for accessibility/ADA and required to show up properly in handbook search.
The page always needs to start with a level 1 heading, which in Markdown starts with a single `#` (the number of hashes matches the level of the heading).

It looks like this page was converted from a Google Docs page. In Google Docs, you achieve this by selecting the heading and using the right heading style:

<img width="278" alt="image" src="https://user-images.githubusercontent.com/10532611/200975757-d7b59d91-74de-48fe-bb57-520ec82582e1.png">

Then make sure to remove any other formatting on the heading:

<img width="224" alt="image" src="https://user-images.githubusercontent.com/10532611/200975828-6663ba49-7808-4108-baa0-ec8447c4c9e4.png">

Then after conversion, this line should simply start with a `#`.